Support.md

```diff
@@ -0,0 +1,75 @@
+- I take my individual growth seriously and realise it is valuable to both myself and to Octopus.
+- I reach out to my peers and leadership group to develop my personal growth plan.
+
+### I am an egoless, team oriented support engineer
+
+- Other team members genuinely want to work alongside me.
+- I happily take responsibility for what I produce.
+- I avoid going dark and I know when to call out for help.
+- I communicate professionally and keep stake holders engaged at all times.
+- I am well aware of all processes of the team and diligently follow them (like maintaining accurate documentation, timely customer updates, and so on).
+- Even when I disagree with something I continue to work enthusiastically, learning from the others in the team.
+- Every venture has mundane tasks, and I work happily and diligently through them.
+- I go out of my way to develop a friendly and professional rapport with my customers and their team members.
+- My contributions to team discussions are net-positive, informative, and helpful.
+
+## Senior Support Engineer
+
+> I am a confident, secure member of the Octopus support team with a proven record of customer support.
+
+### I am trusted to delight our customers with every interaction
+
+- I am proficient with my tools and the core building blocks used by our team.
+- I have a good understanding of more than one area of Octopus, and can confidentaly answer support questions in these areas.
+- I can confidently step outside my comfort zone and adapt quickly to new area of Octopus.
+- I am proficient at qualifying and analysing support and feature requests.
+- I know how to make pragmatic decisions in order to provide the best outcomes for our customers
+- I know when to call out for help and how to do it effectively.
+- I am technically persuasive.
+
+
+### I am a support mentor
+
+- People tend to ask me for my opinion when making support decisions because I have a proven track record of making wise choices.
+- I am fuelling the team's desire to learn, perhaps by mentoring, running brown-bag sessions, sharing useful videos/blogs/papers.
+- I use support response reviews as an opportunity to teach and show others alternate, cleaner ways to delight our customers in an ego-less manner. 
+- I can manage difficult conversations and tactfully challenge others, leaving them with a positive impression of myself and Octopus.
+- I can effectively coach people and pass on my knowledge.
+- I am a confident and proficient teacher of my craft.
+
+### I am focussed on the success of my team and my customers, without the need to boost my ego
+
+- I add value to my team by being a trusted, proactive supporter of my team and its leadership, and by delivering high quality work with a minimum of fuss.
+- I seek to understand the real needs of my customers, and I am comfortable selecting the best approach to best serve those needs (including saying no).
```

Comment:
Alex, you've probably covered this here ☝️ with "real needs", but I remember you telling me once that it's often necessary to anticipate customers needs based on experience with particular issues (and hidden clues in the support ticket, etc). I.e., don't simply answer the stated question and throw the ticket back over the transom knowing it will likely come back again, but take the time to anticipate related issues and provide support before the customers knows it's needed.

Reply:
Nice catch and great wording. I've added that in to the to senior as is, slight variation to support engineer `I am learning to take the time to anticipate related issues and provide support before the customers knows it's needed.`

Properly fixed the oversite/oversight heading as well